web/src/reducers/apd.js

```diff
@@ -64,7 +52,7 @@ const reducer = (state = initialState, action) => {
         {
           byId: {
             [action.data.id]: {
-              ...fromAPI(action.data),
```

Comment:
It was really weird to me that the reducer was responsible for deserializing this data, but only this data - in other places, the data was already deserialized.  So...  I made it consistent.

![The Bobs](https://i.imgflip.com/1q8iud.jpg)